### ISSUE TITLE
pod install failing 'TOCropViewConstants.h' file not found

### ISSUE THREAD

**Issue by amosaxe** [hhunaid/react-native-image-crop-tools#86]
When using this library I am getting '`TOCropViewConstants.h' file not found` while doing `pod install`.  This is similar to the issue https://github.com/ivpusic/react-native-image-crop-picker/issues/1294 in [react-native-image-crop-picker](https://github.com/ivpusic/react-native-image-crop-picker). 

is there a way to use this library with `use_frameworks!` in the pod file?

**Comment by Ahsanali012** [comment]
any updates???

**Comment by amosaxe** [comment]
No. I was using both [react-native-image-crop-picker](https://github.com/ivpusic/react-native-image-crop-picker) and image-crop-tools, and had to pick one between this.

**Comment by Ahsanali012** [comment]
Can you suggest the solution or other library that might help as i think for react-native-crop-tools too there is this same error

**Comment by amosaxe** [comment]
I actually shifted fully to [react-native-image-crop-picker](https://github.com/ivpusic/react-native-image-crop-picker).

**Comment by Ahsanali012** [comment]
But for me I am getting the error TOCropViewControllerTransitioning.h' file not found. I have react native version 0.72.3

**Comment by amosaxe** [comment]
you are getting this error in image-crop-picker as well? @Ahsanali012

**Comment by Ahsanali012** [comment]
Yes its TOCropViewControllerTransitioning.h' file not found . I want to use this Library as I need to show user preview of image for cropping, Anybody have suggestion?

**Comment by amosaxe** [comment]
Actually I am also looking for such cropping library and this is the only possible one i found https://github.com/vemarav/react-native-avatar-crop

**Comment by Ahsanali012** [comment]
but its cropping in circle I dont want to crop in circle

> Actually I am also looking for such cropping library and this is the only possible one i found https://github.com/vemarav/react-native-avatar-crop

**Comment by amosaxe** [comment]
There is option of rect as well

that is rectangle

**Comment by amosaxe** [comment]
leave that, i also tried that package and its lacks a lot of things

**Comment by prachipatel01** [comment]
@Ahsanali012 were u able to find something?

**Comment by Ahsanali012** [comment]
No 

> @Ahsanali012 were u able to find something?
